<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentsWindow</class>
 <widget class="QWidget" name="PaymentsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_payments">
      <attribute name="title">
       <string>Registro de pagos</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label_provider">
           <property name="text">
            <string>Provedor</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="combobox_provider"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_date0">
           <property name="text">
            <string>Fecha</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_value">
           <property name="text">
            <string>Valor</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineedit_value"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_type">
           <property name="text">
            <string>Tipo de pago</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="combobox_type"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_from">
           <property name="text">
            <string>Pagar de</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="combobox_from"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_detail">
           <property name="text">
            <string>Detalle</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QDateEdit" name="dateedit_date">
             <property name="displayFormat">
              <string>dd-MM-yy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_date">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineedit_detail"/>
           </item>
           <item>
            <widget class="QLabel" name="label_voucher">
             <property name="text">
              <string>CE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineedit_voucher">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="button_savepayment">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Registrar Pago</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_provider">
      <attribute name="title">
       <string>Nuevo Proveedor/Beneficiario</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_id">
           <property name="text">
            <string>Identificación</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineedit_id">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_name">
           <property name="text">
            <string>Nombre</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineedit_name"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_phone">
           <property name="text">
            <string>Teléfono</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineedit_phone"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_address">
           <property name="text">
            <string>Dirección</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineedit_address"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="button_saveprovider">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Guardar Proveedor</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
